### PR TITLE
clean README from some Mapboxes

### PR DESCRIPTION
**Pull request by matkoniecz**
links were tested and confirmed to work

PS Is there any CI setup for this repo? Should mapbox CI badge be removed or replaced?

(after this one, #4 #5 #8 are processed I will open more PRs)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,29 +12,29 @@
 </p>
 <br>
 
-# Mapbox Plugins for Android
+# Maplibre Plugins for Android
```
Comment (wipfli) [inline README.md:15]:
```suggestion
# MapLibre Plugins for Android
```

**Review by wipfli — APPROVED** [review]
Thanks

**Comment by wipfli** [comment]
I don't know about CI. Feel free to investigate...

**Comment by matkoniecz** [comment]
I opened #9 for now (I assume that CI is not setup, maybe it should be)

**Pull request merged**